apiserver/stateauthenticator/context.go

```diff
@@ -298,8 +301,5 @@ func newExternalMacaroonAuth(st *state.State, clock clock.Clock, identClient ide
 		Location: idURL,
```

Comment:
There is a comment from axw just above here about storing the key in mongo. Could remove that now?